wpilibc/src/main/native/cppcs/RobotBase.cpp

```diff
@@ -7,10 +7,13 @@
 
```

Comment:
Why did this need to move folders?

Reply:
Because the base folder actually uses the same object files for both shared and static to improve build times, and we needed different behavior for this specific file depending on if built into a shared or static library.